04-onion-routing.md

```diff
@@ -853,7 +853,9 @@ The top byte of `failure_code` can be read as a set of flags:
 * 0x1000 (UPDATE): new channel update enclosed
 
 Please note that the `channel_update` field is mandatory in messages whose
-`failure_code` includes the `UPDATE` flag.
+`failure_code` includes the `UPDATE` flag. It is encoded as only the bytes
+which would appear in the message body, *not* including the `0x0102` message
```

Comment:
Right now eclair supports decoding both cases, but encodes with the `0x0102` message prefix (IIRC one implementation only supported reading that case at some point). I'm happy to change it if everyone supports it today.

Reply:
I believe lnd and we do not use the message type prefix, CLN and y'all do. I'm happy with either, I don't really care, I'm also happy to flip a coin.

Reply:
I'd rather drop the message prefix as you propose, I think it makes more sense without it, I just need to make sure that latest versions of all implementations will know how to decode that.

Reply:
Whether they do or not, no-type-prefix is, from what I've seen, by far the most common version on the network, likely because its what lnd does.